pkg/asset/manifests/content/bootkube/kube-system-configmap-etcd-serving-ca.go

```diff
@@ -0,0 +1,23 @@
+package bootkube
+
+import "text/template"
+
+var (
+	// KubeSystemConfigmapEtcdServingCA  is the constant to represent contents of kube-system-configmap-etcd-serving-ca.yaml file
+	KubeSystemConfigmapEtcdServingCA = template.Must(template.New("kubesystemconfigmapetcdservingca.yaml").Parse(`
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: etcd-serving-ca
+  namespace: kube-system
+  labels:
+    tier: "control-plane"
+    k8s-app: "kube-apiserver"
+    openshift.io/control-plane: "true"
+    openshift.io/component: "api"
+data:
+  ca-bundle.crt: |
+    {{.EtcdCaCert}}
```

Comment:
`EtcdCaCert` is base64encoded, so you can just `ca-bundle.crt: {{.EtcdCaCert}}`

Reply:
thanks!